ui/packages/shared/components/package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "test": "jest --coverage --config ../../../jest.config.js ./src/*",
     "prepublish": "yarn build",
-    "build": "tsc && tailwindcss -o dist/styles.css --minify"
+    "build": "tsc && tailwindcss -o dist/styles.css --minify",
+    "watch": "tsc --watch && tailwindcss -o dist/styles.css --minify"
```

Comment:
I suppose `--watch` does not exit unless it is interrupted, so anything right of `&&` will never run.

`tailwindcss` also has a `--watch` flag, looks like it would need to run in parallel to `tsc`

Reply:
Good catch. I'll modify that

Reply:
Interestingly, after removing everything right of `&&`, Tailwind CSS changes still cause a hot reload of the app. I assume this is because the `dist` folder already contains the Tailwind CSS file itself, and we're merely now just changing classnames in components.

Reply:
No, I think, it just works when you add a class that is already there in the css file. 
It'll break if we add a new class that is not used in any of the source files yet, as tailwind scans the source files and adds styles to the output css only for the classes that are found in the scanned files. 

We should put back the tailwind command for it to work properly with future changes as well.

Reply:
got it. I'll open another PR for that.